doc/rtd/development/packaging.rst

```diff
@@ -0,0 +1,105 @@
+.. _building_and_distribution:
+
+Building and Distribution
+*************************
+
+``Cloud-init`` is distributed as a system package and pre-installed in cloud
+images. Correct behavior depends on the tight integration of system services
+performing system-wide configuration during early boot stages. Because of
+this, cloud-init is not published to PyPI as it is not intended to be consumed
+as a pure-python package. Instead, cloud-init uses meson_ to build all
+dependent data files, system-initialization scripts and global configuration
+files.
+
+The build system used by ``cloud-init`` is `meson`_. ``Cloud-init`` is not
+published to pypi.org because it is not intended to be consumed directly as a
+pure-python library, SDK or API for managing cloud configuration.
+
+``Cloud-init`` correct behavior depends on the tight integration of
+system services performing :ref:`the early boot stages<boot_stages>` and
+system-wide configuration. Upstream uses `meson`_ build backend to support the
+distribution of all dependent data files, system-initialization scripts and
+global configuration files.
```

Comment:
While this is all correct and nicely written, it does duplicate content written elsewhere and isn't strictly necessary to justify the case.

It would be more direct and to the point if this document's opening section  clearly states that the target audience of this documentation page is operating system packagers, and that packages built using the steps on these pages do not produce supported packages.

Reply:
Good thought, I moved the sentence from `Test builds of DEBs...`  up in the intro paragraph `. One should first look to official downstream cloud images containing pre-built packages of``cloud-init`` for your preferred operating system.`  along with a `intended for downstream OS developer audience` clause.